proto/ibc/core/channel/v1/query.proto

```diff
@@ -274,8 +332,8 @@ message QueryUnrelayedAcksRequest {
   string port_id = 1;
   // channel unique identifier
   string channel_id = 2;
-  // list of commitment sequences
-  repeated uint64 packet_commitment_sequences = 3;
+  // list of acknowledgment sequences
+  repeated uint64 packet_ack_sequences = 3;
 }
 
 // QueryUnrelayedAcksResponse is the response type for the
```

Comment:
Personally believe `QueryUnrelayedAcks` should be renamed to `QueryUnreceivedAcks` since this should be sent to original sending chain which is waiting to receive acks

Reply:
Isn't it a RPC sent to the chain writing acknowledgements? see [previous discussion](https://github.com/cosmos/cosmos-sdk/pull/7320#issuecomment-693705412)

Reply:
I was wrong back then. Definitely this RPC is meant to go to the chain receiving acknowledgements since that is the original sender chain which still has packet commitments in it if ack hasn't been received

Reply:
Renamed it along with docs, please double check the implementation to confirm my thinking here. 

QueryUnreceivedAcks is being passed in a list of ack sequences and it will then check if the executing chain still has a packet commitment for that sequence. If it does, then the sequence is added to unreceived acks. Thus, the executing chain must be the original sender since it stores the packet commitments

Reply:
Sorry for the flipflop!! Hopefully the last change!